config/generic-bigtreetech-skr-mini-e3-v3.0.cfg

```diff
@@ -107,6 +107,9 @@ pin: PB15
 [fan]
 pin: PC6
 
+[filament_switch_sensor e0_sensor]
```

Comment:
Please comment these new lines, as a new user may not have a filament sensor, but allows them to have it as an option if needed.

Thanks
James

Reply:
Hi James,

I have done as requested, but in addition I have created a commented out section for a bed probe.  Both of these I have tested on this board.

Kind regards,

ExplicitChimp